cumulusci/tasks/salesforce/tests/test_EnsureRecordTypes.py

```diff
@@ -151,6 +154,9 @@ def test_no_business_process_where_unneeded(self):
         self.assertNotIn("stage_name", task.options)
 
     def test_generates_record_type_and_business_process(self):
+        """
+        Asserts Record Type Description is optional.
+        """
```

Comment:
Please don't add docstrings to test methods -- IIRC the test runner is a bit weird and will show them instead of the name of the method. Use a comment starting with `#` instead.

Reply:
Will convert.  Thank you!

Reply:
Done.